lib/Jsrt/Jsrt.cpp

```diff
@@ -4293,11 +4293,28 @@ _ALWAYSINLINE JsErrorCode CompileRun(
 {
     PARAM_NOT_NULL(scriptVal);
     VALIDATE_JSREF(scriptVal);
+    PARAM_NOT_NULL(sourceUrl);
```

Comment:
We don't expect source url to be null at all places, RunScriptCore have PARAM_NOT_NULL(sourceUrl);